include/tatami/isometric/unary/math_helpers.hpp

```diff
@@ -58,6 +58,52 @@ struct DelayedAbsHelper {
      */
 };
 
+/**
+ * @brief Take the signum of a matrix entry.
```

Comment:
Is `signum` a real word? Why not just say `sign`?

Reply:
Signum is a latin word. Changed to sign